pgcommitfest/commitfest/views.py

```diff
@@ -86,14 +86,36 @@ def me(request):
     if patch_list.redirect:
         return patch_list.redirect
 
+    # Get stats related to user for current commitfest
+    curs = connection.cursor()
+    curs.execute(
+        """SELECT
+            ps.status, ps.statusstring, count(*)
+        FROM commitfest_patchoncommitfest poc
+        INNER JOIN commitfest_patchstatus ps ON ps.status=poc.status
+        LEFT JOIN commitfest_patch_authors pa ON pa.patch_id=poc.patch_id
+        LEFT JOIN commitfest_patch_reviewers pr ON pr.patch_id=poc.patch_id
```

Comment:
This will inflate counts if a patch has more than one author or reviewer. Instead of doing this join we should use the same WHERE filter we use for the display list:
```
            EXISTS (
                SELECT 1 FROM commitfest_patch_reviewers cpr WHERE cpr.patch_id=p.id AND cpr.user_id=%(self)s
            ) OR EXISTS (
                SELECT 1 FROM commitfest_patch_authors cpa WHERE cpa.patch_id=p.id AND cpa.user_id=%(self)s
            ) OR p.committer_id=%(self)s""")
```